com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/Targets/BuiltInLitSubTarget.cs

```diff
@@ -58,6 +62,24 @@ public override void Setup(ref TargetSetupContext context)
             }
         }
 
+        public override void ProcessPreviewMaterial(Material material)
+        {
+            // copy our target's default settings into the material
+            // (technically not necessary since we are always recreating the material from the shader each time,
+            // which will pull over the defaults from the shader definition)
+            // but if that ever changes, this will ensure the defaults are set
+            material.SetFloat(Property.SpecularWorkflowMode(), (float)workflowMode);
+            //material.SetFloat(Property.SG_CastShadows, target.castShadows ? 1.0f : 0.0f);
```

Comment:
no shadows?

Reply:
Right now it's always on. There's no option yet. I'd have to investigate how to do that, especially for the receive shadows part.